commands/createfixpullrequests.go

```diff
@@ -261,18 +251,11 @@ func (cfp *CreateFixPullRequestsCmd) createFixVersionsMap(scanResults []services
 	return fixVersionsMap, nil
 }
 
-func (cfp *CreateFixPullRequestsCmd) addVulnerabilityToFixVersionsMap(vulnerability *formats.VulnerabilityOrViolationRow, fixVersionsMap map[string]*FixVersionInfo) error {
+func (cfp *CreateFixPullRequestsCmd) addVulnerabilityToFixVersionsMap(vulnerability *formats.VulnerabilityOrViolationRow, fixVersionsMap map[string]*packagehandlers.FixVersionInfo) error {
 	if len(vulnerability.FixedVersions) == 0 {
 		return nil
 	}
-
-	fixVulnerability, err := cfp.shouldFixVulnerability(vulnerability)
-	if err != nil {
-		return err
-	}
-	if !fixVulnerability {
-		return nil
-	}
+	isDirectDependency := cfp.IsDirectDependency(vulnerability)
```

Comment:
The above variable is redundant. Let's call the `cfp.IsDirectDependency(vulnerability)` function below when needed.